src/Scanner.ts

```diff
@@ -119,6 +127,7 @@ class Scanner {
 
   private addTag(rawContent: string, name: string, restContent = '') {
     const cleanName = name.replace(reNonchar, '').toLowerCase();
+    const attributes = this.matchAttributes(rawContent);
```

Comment:
I originally intended `restContent` to contain "raw attributes", but never passed it there. When the attributes are implemented, the `restContent` can be removed from everywhere